locales/ko.schema.json

```diff
@@ -1191,7 +1191,7 @@
           "settings": {
             "show_dynamic_checkout": {
               "label": "동적 결제 버튼 표시",
-              "info": "스토어에서 사용 가능한 결제 방법을 사용하면 고객이 PayPal 또는 ApplePay처럼 원하는 옵션을 볼 수 있습니다. [자세히 알아보기](https://help.shopify.com/manual/using-themes/change-the-layout/dynamic-checkout)"
+              "info": "스토어에서 사용 가능한 결제 방법을 사용하면 고객이 PayPal 또는 ApplePay처럼 원하는 옵션을 볼 수 있습니다. [자세히 알아보기](https:\/\/help.shopify.com\/manual\/using-themes\/change-the-layout\/dynamic-checkout)"
```

Comment:
@martinamarien I remember we ran into this issue on Kai's PR. Do you know why this is being added?